Description: Avoid autotools messing with debian subdir
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2006-08-19

--- a/Makefile.am
+++ b/Makefile.am
@@ -3,7 +3,7 @@
 AUTOMAKE_OPTIONS = foreign dist-zip
 ACLOCAL_AMFLAGS = -I m4
 
-SUBDIRS = src conf debian doc web admin win32 examples
+SUBDIRS = src conf doc web admin win32 examples
 
 EXTRA_DIST = HACKING m4/acx_pthread.m4 m4/ogg.m4 \
     m4/theora.m4 m4/vorbis.m4 m4/speex.m4 \
--- a/configure.in
+++ b/configure.in
@@ -150,7 +150,7 @@
 AC_SUBST(HAVE_KATE)
 AC_SUBST(KATE_LIBS)
 
-AC_OUTPUT([Makefile conf/Makefile debian/Makefile src/Makefile src/avl/Makefile
+AC_OUTPUT([Makefile conf/Makefile src/Makefile src/avl/Makefile
 src/httpp/Makefile src/thread/Makefile src/log/Makefile
 src/net/Makefile src/timing/Makefile doc/Makefile web/Makefile
 admin/Makefile win32/Makefile win32/res/Makefile examples/Makefile])
